### ISSUE TITLE
Report statment and states in vhdl 

### ISSUE THREAD

**Issue by aniketabhiraj2004** [ghdl/ghdl-yosys-plugin#198]
Is their is a way to see report statment and the current state by name. Because cxxrtl ignore the report statment and convert all state into binary.

**Comment by tgingold** [comment]
No.  I am pretty sure cxxrtl can only work on a netlist and report statements are not synthesizable.

You don't you simply simulate with ghdl ?
